### ISSUE TITLE
Update wiki with relevant resources

### ISSUE THREAD

**Issue by koustuvsinha** [iem-devs/datascience#1]
Our wiki should have all relevant links, tutorials and blog links related to Data Science.

**Comment by ashunigion** [comment]
the wiki link is broken :-(

**Comment by ashunigion** [comment]
@koustuvsinha bhaiya i have made the changes to wiki (https://github.com/ashunigion/datascience/wiki), please review them and edit the main wiki page.

**Comment by koustuvsinha** [comment]
pretty good.. can you directly update the wiki here?

**Comment by ashunigion** [comment]
yes bhaiya! but i dont know how to submit pull request for wiki, i have tried a lot of resources to learn, but have failed. If you could help me with the steps!

**Comment by koustuvsinha** [comment]
No need for a PR for the wiki, just update it directly!

**Comment by ashunigion** [comment]
bhaiya, I dont see the edit option on the page(like i see in my own repositories). So i have tried the following.
1. i cloned the wiki
2. added the updated files
3. pushed it..(but i got his error)

   ` hrcule@shrishti:~/iem-dev/datascience.wiki$ git push`
    `Username for 'https://github.com': ashunigion`
    `Password for 'https://ashunigion@github.com':` 
    `remote: Permission to iem-devs/datascience.wiki.git denied to ashunigion.`
    `fatal: unable to access 'https://github.com/iem-devs/datascience.wiki.git/': The requested URL` 
    `returned error: 403`

But bhaiya these steps work for the repositories that I own.

Please suggest the __necessary changes__ in my steps

**Comment by koustuvsinha** [comment]
Ah, I think you weren't in the team with `Write` permissions. Added you, can you try now?

**Comment by ashunigion** [comment]
updated it! :smile:

**Comment by ashunigion** [comment]
Bhaiya i have repository that have mini projects which i developed while learning [Python libraries for Data-science](https://github.com/ashunigion/into-ML-with-Python) should i create a repository in iem-devs and push them.
